src/extent.jl

```diff
@@ -14,12 +14,20 @@ GI.ismeasured(::GI.AbstractPointTrait, ::Point) = false
 GI.ismeasured(::GI.AbstractPointTrait, ::Union{PointM,PointZ}) = true
 
 # extent
-function GI.extent(x::AbstractShape)
+GI.extent(::GI.PointTrait, p::Union{Point,PointM}) =
+    Extents.Extent(X=(p.x, p.x), Y=(p.y, p.y))
+GI.extent(::GI.PointTrait, p::PointZ) =
+    Extents.Extent(X=(p.x, p.x), Y=(p.y, p.y), Z=(p.y, p.y))
```

Comment:
```suggestion
    Extents.Extent(X=(p.x, p.x), Y=(p.y, p.y), Z=(p.z, p.z))
```

Reply:
lol, you can tell this was slightly rushed

Reply:
It has a test now